<!DOCTYPE html>
<html>
<head>
  <script type="text/javascript" async
    src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/MathJax.js?config=TeX-MML-AM_CHTML">
  </script>
  <style>
    /* Center-align the content within the div */
    .centered {
      text-align: center;
    }
    
    blockquote {
      margin-left: 20px; /* Adjust the value to set the desired indentation */
    }
  </style>
</head>
<body>

<h5>Summarized Info</h5>
<ul>
  <li><strong>Tissue:</strong> Refers to the tissue with the highest expression level, based on two or more databases including GEO, GTEx, and TCGA. If the same tissue is identified as having the highest expression in two or more databases, it is referred to as that tissue. If different tissues are identified as having the highest expression levels across the three databases, the tissue that gives the highest value of (expression level of the tissue with the highest expression) / (expression level of the tissue with the second highest expression) is selected and displayed, according to the described rule.</li>
  <li><strong>TS Score:</strong> Quantifies the degree of tissue-specific expression. We define genes with values equal to or greater than 1.5 as tissue-specific genes. The TS score is calculated by examining Normalized Foldchange (NFC) and Normalized Tau (NTau) across all three databases and then averaging the results. The TS score ranges from -1.5 to 5.2. The NFC and Ntau values are explained in the TS Gene Info section below. Please refer to the image below for distribution visualization.</li>
  <li><strong>Consecutive score:</strong> A measure of how consistently a gene is expressed. We assigned a score of 1 to genes with expression levels in the top 75% within samples out of a total of 46,813 samples, and a score of 0 otherwise. We then, we combined and normalized the binary distribution of all genes and normalized it. The score ranges from -0.8 to 2.35. Please refer to the image below for a visualization of the distribution.</li>
  <li><strong>Up/Down Regulated Tissue:</strong> The criteria for "regulated" are based on the tissue with the highest or lowest expression level or the average expression across all tissues. Genes with expression levels more than 5-fold higher are categorized as tissue upregulated genes, while genes with expression levels more than 0.2-fold lower are categorized as downregulated genes. A gene may associated with more than one regulated tissue.</li>
  <li><strong>HKG:</strong> Please refer to the HouseKeeping INFO section for a detailed explanation of the parameters. We pre-filtered for genes that show a difference of 4 or more in expression levels between the tissue with the highest expression and the tissue with the lowest expression. We classify genes with a coefficient of variation (CV) in the lower 25% as housekeeping genes (HKG).</li>
</ul>

<h5>House Keeping Gene Info</h5>
<ul>
  <li><strong>CV:</strong> <a href="https://en.wikipedia.org/wiki/Coefficient_of_variation">Coefficient of variation</a>. The coefficient of Variation (CV) is defined as the ratio of the standard deviation. We chose CV instead of standard deviation, because the library sizes of each sample are not equal. CV values range from 0 to 1. A value closer to 0 indicates a more housekeeping pattern.</li>
  <li><strong>Foldchange:</strong> For a house keeping gene, this is the ratio of the average expression level in the tissue with the highest expression to the average expression level in the tissue with the lowest expression. We use fold change as a pre-filtering criterion.</li>
  <li>We pre-filtered genes with an average fold change (FC) of 4 or higher and defined genes with an average coefficient of variation (CV) in the lower 25% across the three databases as housekeeping Genes (HKG). You can adjust the criteria on the download page to customize your downloads.</li>
</ul>

<h5>Tissue Specific Gene Info</h5>
<ul>
  <li><strong>NTau:</strong> Normalized <a href="https://academic.oup.com/bib/article/18/2/205/2562739">Tau</a>. Tau calculated in this equation :</li>
</ul>

<div class="centered">
  <blockquote>
    <p>\(x_i\) is the expression of the gene in tissue \(i\).</p>
    <p>\(n\) is the number of tissues.</p>
  </blockquote>
  <p><img src="https://latex.codecogs.com/svg.latex?\tau=1-\frac{1}{n-1}\sum_{i=1}^{n}\hat{x}_i" alt="\tau Equation" /></p>
  <p><img src="https://latex.codecogs.com/svg.latex?\hat{x_i}=\frac{x_i}{\max(x_i)}" alt="\hat{x_i} Equation" /></p>
</div>

<ul>
  <li><strong>NFC:</strong> Normalized fold change, fold change is the ratio of the average expression level in the tissue with the highest expression to the average expression level in the tissue with the second highest expression. For all genes, FC was examined for each database, and NFC was calculated by adjusting with Z-scores.</li>
  <li><strong>TS score:</strong> Mean of <strong>NTau</strong> and <strong>NFC</strong></li>
</ul>

<h5>Promoter Info</h5>
<ul>
  <li>All promoter information obtained from <a href="https://epd.expasy.org/epd/EPDnew_database.php">EPD new</a></li>
  <li><strong>Promoter type:</strong> Three categories of promoters are identified, indicating the diversity of transcription initiation patterns in eukaryotes. <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005144">Dreos et al., 2016</a></li>
  <li><strong>Motifs:</strong> GC box, Inr region, TATA box, CCAAT box are defined.</li>
  <li><strong>Promoter sequence:</strong> A sequence ranging from -50bp to 10bp relative to the transcript region is shown.</li>
</ul>

</body>
</html>



